x-pack/plugins/actions/server/routes/create.ts

```diff
@@ -57,7 +57,7 @@ export function createRoute(server: Hapi.Server) {
       },
     },
     async handler(request: CreateRequest) {
-      const actionsClient = request.getActionsClient();
+      const actionsClient = request.getActionsClient!();
```

Comment:
It's not guaranteed the function `getActionsClient` exists on the request. The exclamation point tells TypeScript to ignore the error of it possibly being `undefined`. In order for it to exist you need the `actions` plugin enabled. It's a bit weird in this context since we are within the plugin itself but the TypeScript configuration for `getActionsClient` is done at a global level.

Reply:
Maybe a dumb question, but wouldn't the next line fail if that method doesn't exist?  Or does TS just skip execution from the error onwards?

Reply:
The next line would fail if the method doesn't exist. But it should be there since the plugin requires `actions` plugin and disables itself if anything it requires is disabled.